spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/stores/ConfigStore.java

```diff
@@ -94,8 +94,8 @@ public void validateAndInit() {
         if (StringUtils.hasText(connectionString)) {
             String endpoint = (new Connection(connectionString)).getEndpoint();
             try {
-                URI uri = new URI(endpoint);
-                this.name = uri.getHost().split("\\.")[0];
+                new URI(endpoint);
```

Comment:
Wondering where we use this uri?

Reply:
Just validation. new URI checks if its valid or not, but we don't need the URI object. I will add a comment saying as much.